src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+const successEl = (message) =>
+  `<div data-qa="notification" class="success">${message}</div>`;
+const errorEl = (message) =>
+  `<div data-qa="notification" class="error">${message}</div>`;
+
+const displayMessage = (message) => {
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('beforeend', message);
+};
+
+const promise1 = Promise.race([
+  new Promise((resolve) => {
+    const handleClick = () => {
+      resolve(successEl('First promise was resolved'));
+      document.removeEventListener('click', handleClick);
+    };
+
+    document.addEventListener('click', handleClick);
+  }),
+  new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(errorEl('First promise was rejected'));
+    }, 3000);
+  }),
+]);
+
+const promise2 = new Promise((resolve) => {
+  const handleClick = (e) => {
+    if (e.button === 0 || e.button === 2) {
+      resolve(successEl('Second promise was resolved'));
+      document.removeEventListener('click', handleClick);
+      document.removeEventListener('contextmenu', handleClick);
```

Comment:
Same issue as above. You are adding and removing event listeners with the same function. It's better to define the function separately and then use it as a handler.